packages/@okta/vuepress-site/docs/reference/api/risk-providers/index.md

```diff
@@ -0,0 +1,307 @@
+---
+title: RiskProviders
+category: management
+---
+
+# RiskProviders API
+
+The Okta RiskProviders API provides the ability to manage the risk providers within Okta.
+
+## Get Started
```

Comment:
As an aside, I think the two separate API pages could be combined into one API page called "Risk" or "Risk APIs", just an fyi

Reply:
I wanted to not combine into a single `Risk` or `Risk APIs` since there are policy APIs that we plan to introduce for `Risk` too which are different from these APIs which are related to the third party risk events.